history/characters/310000_patron_characters.txt

```diff
@@ -6,7 +6,7 @@
 	culture=lordaeronian religion=holy_light
 	martial=8 diplomacy=4 stewardship=7 intrigue=5 learning=4
 	add_trait=skilled_tactician
-	add_trait=patron
+	add_trait=patron effect={purge_patron_trait_character_effect=yes}
```

Comment:
![image](https://user-images.githubusercontent.com/46384992/70823814-38c9cc00-1dfa-11ea-86f5-2e374523f8e4.png)
Isn't it easier to make `on_startup` event?

Reply:
No